### PR TITLE
Issue #2162: Change Front page to Home page in core.

### LINKED ISSUE #2162: [UX] Change `Front page` to `Home page` in core
We have a lot of places in core that refer to the home page of the site as a `Front page`. This is a bit of a drupalism and I would love to see us move to calling it the `Home page` consistently. That, however, will be a lot of work. Here are some of the places we need to update:

- [x] #3855 | [UX] Update "Front page" to "Home page" on the Site information page
- [ ] The silly `<front>` placeholder
- [x] After running update.php link
- [x] Layout "Is this the front page." condition
- [x] Redirect module validation
- [x] Token for `[site:url]` description
- [ ] ~The API-level config variable `config_get('system.core', 'site_frontpage');`~
- [x] more TBD

Spun off from https://github.com/backdrop/backdrop-issues/issues/1042#issuecomment-246280693

---

~~PR by @jenlampton: https://github.com/backdrop/backdrop/pull/1544~~
PR by @klonos: https://github.com/backdrop/backdrop/pull/2729 (excludes `system.admin.inc`, which is done in #3855 / https://github.com/backdrop/backdrop/pull/2728)

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2162

### CODE REVIEW

File: core/modules/system/system.admin.inc [hunk 1]
```diff
@@ -1203,14 +1203,14 @@ function system_site_information_settings($form, &$form_state) {
...
+    '#title' => t('Home page'),
   );
   $form['front_page']['site_frontpage'] = array(
     '#type' => 'textfield',
-    '#title' => t('Default front page'),
+    '#title' => t('Default home page'),
     '#default_value' => ($site_config->get('site_frontpage') != 'user' ? backdrop_get_path_alias($site_config->get('site_frontpage')) : ''),
     '#size' => 40,
-    '#description' => t('Optionally, specify a relative URL to display as the front page.  Leave blank to display the default content feed.'),
+    '#description' => t('Optionally, specify a relative URL to display as the home page.  Leave blank to display the default content feed.'),
```
Comment (quicksketch) [inline core/modules/system/system.admin.inc:1213]: While changing this string, let's remove the double-space at the end of the sentence. Also is "Leave blank to display the default content feed." true? What happens when this is blank now?

Comment (jenlampton) [reply core/modules/system/system.admin.inc:1213]: I don't know. That string went in with the "disable /node" PR so I assumed it was correct.

Comment (quicksketch) [reply core/modules/system/system.admin.inc:1213]: In any case let's remove the double-space.

**Comment by klonos** [comment]
Should we close this in favor of the newest https://github.com/backdrop/backdrop/pull/2728 @jenlampton??

**Comment by serundeputy** [comment]
I think @klonos is right and I'm going to close this in favor of https://github.com/backdrop/backdrop/pull/2728

feel free to  re-open if necessary.

**Pull request closed without merging**